packages/flutter_tools/test/commands.shard/hermetic/run_test.dart

```diff
@@ -51,6 +52,11 @@ void main() {
       mockUnstableFlutterVersion = MockFlutterVersion(isStable: false);
     });
 
+    setUp((){
+      mockStdio = MockStdio();
```

Comment:
nit. you can use Dart's cascade operator to make these two lines a single declaration:

```dart
mockStdio = MockStdio()..stdout.terminalColumns = 80;
```

Reply:
TIL! Thanks and done.